pkg/steps/pod.go

```diff
@@ -44,44 +50,9 @@ func (s *podStep) Inputs(ctx context.Context, dry bool) (api.InputDefinition, er
 func (s *podStep) Run(ctx context.Context, dry bool) error {
 	log.Printf("Executing %s %s", s.name, s.config.As)
 
-	containerResources, err := resourcesFor(s.resources.RequirementsForStep(s.config.As))
```

Comment:
Moving all this makes it hard to see your diff, please don't do that

Reply:
Regardless -- break it out into a separate commit or don't do it, it's not germane to the changes you are making here.